lib/broccoli/angular2-app.js

```diff
@@ -227,6 +176,166 @@ Angular2App.prototype.index = function () {
 
   return configReplace(index, {
     files: [htmlName],
-    patterns: this._configReplacePatterns()
+    patterns: this._getReplacePatterns()
+  });
+};
+
+
+/**
+ Returns the source root dir tree
```

Comment:
When using the `/** ... */` notation, every line should be prefixed with ` * `.

Like this:

```
/**
 * Returns the source root dir tree.
 * @private
 * @return {Tree} The tree containing all source files.
 */
```

Please do this for all the comments that match this format. Also, end your sentences with a period (`.`).